packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -134,7 +107,6 @@ ApiReportGenerator.generateReviewFileContent =
     ...moreArgs: any[]
   ) {
     const program = collector.program as Program;
-
     // The purpose of this override is to allow the @ignore tag to be used to ignore warnings
     // of the form "Warning: (ae-forgotten-export) The symbol "FooBar" needs to be exported by the entry point index.d.ts"
```

Comment:
do we want to keep this `@ignore` tag? if we can allow those warning anyway not sure if is giving to much value?
we have only 18 ignore tags atm

Reply:
Yep I'd say we still need it, unless we can selectively ignore the forgotten export warning for a single symbol? The forgotten export warning is very important to handle, and the `@ignore` tag is an escape hatch for when we really don't want to export the type.